CoreScriptingView/src/au/gov/asd/tac/constellation/views/scripting/ScriptingExecutePlugin.java

```diff
@@ -67,7 +67,7 @@
  * @author cygnus_x-1
  */
 @ServiceProvider(service = Plugin.class)
-@PluginInfo(pluginType = PluginType.NONE, tags = {"LOW LEVEL"})
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```

Comment:
:+1:

Reply:
did this need action or its a good thing?

Reply:
A good thing @aldebaran30701 to say thanks for catching it.